arviz/tests/base_tests/test_data.py

```diff
@@ -323,6 +324,26 @@ def test_addition(self):
         fails = check_multiple_attrs(test_dict, new_idata)
         assert not fails
 
+    @pytest.mark.parametrize("inplace", [True, False])
+    def test_extend_xr_method(self, inplace):
+        data = np.random.normal(size=(4, 500, 8))
+        dataset = from_dict(
```

Comment:
i'd use `idata` or `data`, `dataset` is very confusing as it seems like this is an xarray dataset